Updating matched-data-cli tool instructions (#7094)
cloudflare/cloudflare-docs: content/cloudflare-one/policies/filtering/http-policies/data-loss-prevention/payload-logging.md

To generate a public/private key pair:

> 1. [Download](https://github.com/cloudflare/matched-data-cli) the Cloudflare `matched-data-cli` tool on GitHub by selecting **Code** > **Download ZIP**.

Reviewer: Why are we instructing users to download the source code instead of the pre-built binaries?

Author: Using the prebuilt binary in the WAF instructions was causing users to get an error message "“matched-data-cli” cannot be opened because the developer cannot be verified." I wanted to get up some instructions on how to get around it over the holidays for anyone who wanted to test while looking into the verified issue.

Reviewer 2: I can adjust our docs. Let's add a link to the macOS documentation on executing unsigned apps as troubleshooting help.

> 1. [Download](https://github.com/cloudflare/matched-data-cli) the Cloudflare `matched-data-cli` tool on GitHub using the **<> Code** button and **Download Zip**.

Reviewer 3: ```suggestion
1. [Download](https://github.com/cloudflare/matched-data-cli) the Cloudflare `matched-data-cli` tool on GitHub by selecting **Code** > **Download ZIP**.
```

2. Extract the content of the downloaded `.zip` file to a local folder.

3. Open a terminal and navigate to the local folder containing the `matched-data-cli` tool.

    ```sh

>     $ cd matched-data-cli-master/target/debug

Reviewer: This does not exist?

3. Select **Decrypt Payload Log**.

> 4. Enter your private key and select **Decrypt**. You will see the DLP Profile ID and the decrypted payload.

Reviewer 3: ```suggestion
4. Enter your private key and select **Decrypt**.  You will see the [ID of the matched DLP Profile](https://developers.cloudflare.com/api/operations/dlp-profiles-list-all-profiles) and the decrypted payload.
```

3. Select **Decrypt Payload Log**.

4. Enter your private key and select **Decrypt**. 

> You will see the [ID of the matched DLP Profile](https://developers.cloudflare.com/api/operations/dlp-profiles-list-all-profiles) and the decrypted payload.

Reviewer 3: Need to confirm that the ID shown in the UI is the `id` field in this API.

Author: Just confirmed, profile id is the `id` field in API

> You will see the [ID of the matched DLP Profile](https://developers.cloudflare.com/api/operations/dlp-profiles-list-all-profiles) and the decrypted payload.

Reviewer 3: ```suggestion
You will see the [ID of the matched DLP Profile](https://developers.cloudflare.com/api/operations/dlp-profiles-list-all-profiles) followed by the decrypted payload.
```
